libs/designsystem/dropdown/src/dropdown.component.ts

```diff
@@ -250,7 +250,7 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
     return this._kirbyItemsSlotted;
   }
 
-  private itemClickUnlisten: EventListenerDisposeFn[] = [];
+  private disposeItemClicks: EventListenerDisposeFn[] = [];
```

Comment:
I'd prefer this naming for clarity throughout the files. Thoughts?
```suggestion
  private disposeItemClickListeners: EventListenerDisposeFn[] = [];
```

Reply:
I like that! Its the listener that is disposed and not the event itself